use-location.js

```diff
@@ -1,5 +1,10 @@
 import { useEffect, useRef, useState, useCallback } from "./react-deps.js";
 
+const eventPopState = "popstate"
+const eventPushState = "pushstate"
+const eventReplaceState = "replaceState"
```

Comment:
Should we maybe rename these vars to uppercase? Like:
```suggestion
const REPLACE_STATE_EVENT = "replaceState"
```
?

Reply:
While I understand you concern, I believe some users might have [eslint rule](https://eslint.org/docs/rules/camelcase) for the naming convention:
- "always"

We would make it a pain for them, while if the vars are camelCase, there is a less probability of pissing someone off :)